### ISSUE TITLE
Puppeteer

### ISSUE THREAD

**Issue by deabreu33** [elpatronaco/pccomponentes-buy-bot#45]
Hola amigos, 

Me sale el siguiente error, me podrian ayudar? Gracias de antemano

![image](https://user-images.githubusercontent.com/74322595/141006371-e6509a82-b56a-4259-aeac-dc7853e679c5.png)

**Comment by Rcusa** [comment]
ve al login.js y cambia el data-cy por name, y añades los inputs con el nombre del html

**Comment by deabreu33** [comment]
> ve al login.js y cambia el data-cy por name, y añades los inputs con el nombre del html

Vale a ver si entendí, el "input[data-cy='email']" lo cambio por "input[data-cy='name']"??? y los de los html en que archivo están? Gracias

**Comment by Rcusa** [comment]
No, data-cy es el atributo del html, tienes que buscar en el html de la pccomponentes otro atributo para llamarlo, ya que pccomponentes ha modificado su web, quedaría tal que así  "input[name='email']" y  "input[name='password']"

**Comment by deabreu33** [comment]
> No, data-cy es el atributo del html, tienes que buscar en el html de la pccomponentes otro atributo para llamarlo, ya que pccomponentes ha modificado su web, quedaría tal que así "input[name='username']" y "input[name='password']"

Muchas gracias amigo, solucionado!
